stable/datadog/templates/datadog-yaml-configmap.yaml

```
@@ -37,7 +37,14 @@ data:
       enabled: false
       apm_non_local_traffic: true
 
-    # Use java cgroup memory awareness
+    {{- with $v := splitn "." 2 .Values.image.tag }}
+    {{ if and (ge $v._0 "1") (ge $v._1 "15") }}
```

Ain't this supposed to be compared to 6.15 instead of 1.15?

```suggestion
    {{ if and (ge $v._0 "6") (ge $v._1 "15") }}
```

---

Yes, sorry I botched my commit when squashing stuff 🤦‍♂